Connections-ng-app/src/app/reset/reset.component.ts

```diff
@@ -69,6 +70,20 @@
             err=> {alert("Sorry an error occured");
           });
         }
+
```

Comment:
You are repeating the same code. remember (DRY) Dont Repeat Yourself and reuse the code, or keep the code at one place and use it at places wherever it is being used.

Reply:
Hello,
Taking up this as the next Task.Apologies.
Meanwhile,Please review the rest of the code.
Thanks